main_routers/system_router.py

```
@@ -0,0 +1,538 @@
+        return JSONResponse(content={"success": False, "error": "未找到小于1MB的预览图片文件"})
+        
+    except Exception as e:
+        logger.error(f"查找预览图片文件失败: {e}")
+        # 发生异常时不泄露详细信息
+        return JSONResponse(content={"success": False, "error": "服务器内部错误"}, status_code=500)
+
+# 辅助函数
+
+@router.get('/api/proxy-image')
+async def proxy_image(image_path: str):
+    """代理访问本地图片文件，支持绝对路径和相对路径，特别是Steam创意工坊目录"""
```

Ruff (0.14.8)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)